PHPfiles/Constants.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * A class to hold Theme related constants.
+ */
+class ThemeConstants {
+    /** Table header background color. */
+    const TableHeaderBGColor = "#D9EDF7";
```

Comment:
what does "const" mean?  would you call this constant like this, " ThemeConstants::TableHeaderBGColor " ?

Reply:
Yes.